pom.xml

```diff
@@ -107,7 +107,17 @@
             <artifactId>lombok</artifactId>
             <version>1.16.6</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.nimbusds</groupId>
+            <artifactId>nimbus-jose-jwt</artifactId>
+            <version>4.12</version>
```

Comment:
They have a new version already: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22nimbus-jose-jwt%22

(i.e. we can upgrade Provider too)